nx-dev/nx-dev/pages/ai/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { ReactNode, useState } from 'react';
+import { DocumentationHeader } from '@nx/nx-dev/ui-common';
+import { useNavToggle } from '../../lib/navigation-toggle.effect';
+import { renderMarkdown } from '@nx/nx-dev/ui-markdoc';
+import {
+  getStringFromStream,
+  nxDevDataAccessAi,
+} from '@nx/nx-dev/data-access-ai';
+
+export default function AiDocs(): JSX.Element {
+  const { toggleNav, navIsOpen } = useNavToggle();
+
+  const [finalResult, setFinalResult] = useState('' as ReactNode);
+  const [error, setError] = useState(null);
+  const [query, setSearchTerm] = useState('');
+  const [loading, setLoading] = useState(false);
+
+  const handleSubmit = async () => {
+    setLoading(true);
+    setFinalResult('');
+    try {
+      const response = await nxDevDataAccessAi(query);
+      const completeText = (await getStringFromStream(response)) as string;
+      setFinalResult(renderMarkdown(completeText, { filePath: '' })?.node);
+      setLoading(false);
+    } catch (error) {
+      console.error(error);
+      setError(error);
+      setLoading(false);
+    }
+  };
+
+  return (
+    <div id="shell" className="flex h-full flex-col">
```

Comment:
Since the page will be release darkly, can you make sure it won't be indexed until we are ready?
```
<NextSeo
  noindex={true}
  robotsProps={{
    nosnippet: true,
    notranslate: true,
    noimageindex: true,
    noarchive: true,
    maxSnippet: -1,
    maxImagePreview: 'none',
    maxVideoPreview: -1,
  }}
/>
```
That should deactivate stuff while working on the feature.